InstagramFeed/Base.lproj/Main.storyboard

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="7702" systemVersion="14C109" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" initialViewController="Yfp-Nc-raC">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="6254" systemVersion="14D136" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" initialViewController="Yfp-Nc-raC">
```

Comment:
Haha I did update to the latest version today so idk what's going on



—
Sent from Mailbox

On Fri, May 8, 2015 at 4:22 PM, Neha Sharma <notifications@github.com>
wrote:

>> @@ -1,7 +1,7 @@
>>  <?xml version="1.0" encoding="UTF-8" standalone="no"?>
>> -<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="7702" systemVersion="14C109" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" initialViewController="Yfp-Nc-raC">
>> +<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="6254" systemVersion="14D136" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" initialViewController="Yfp-Nc-raC">
> Uh oh, is this just your super old Xcode version? :P 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/n23sharm/InstagramFeed/pull/1/files#r29984617